src/evaluate.jl

```diff
@@ -2,13 +2,21 @@
 # :input refers to input argument and numbers to the derivative order
 const ARGUMENTS = Dict(
     :lda => [[:zk], [:vrho], [:v2rho2], [:v3rho3], [:v4rho4], ],
+    :hyb_lda => [[:zk], [:vrho], [:v2rho2], [:v3rho3], [:v4rho4], ],
```

Comment:
I don't like this repetition. Perhaps you can just do in a second step (i.e. below the definition of the constant):
```ARGUMENTS[:hyb_lda] = ARGUMENTS[:lda]``` and similarly for the other things.